### PR TITLE
Simplified configurator version checking

### PR DESCRIPTION
**Pull request by blckmn**
As we now have the build API we can perform the version checking server side, and calculate if there is a new version available.

If there is no contact with the build API then the configurator will simply continue to load.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]
Do I understand this correctly. It's just to check what's the latest live release right?

**Pull request merged**
